src/layout/MainLayout.tsx

```diff
@@ -1,13 +1,17 @@
 import { Outlet } from "react-router-dom";
-import HeaderTopNav from "../components/organism/HeaderTopNav";
+import EmptyFiller from "../components/atom/EmptyFiller";
 import SeoHelmet from "../components/atom/SeoHelmet";
+import Footer from "../components/organism/Footer";
+import HeaderTopNav from "../components/organism/HeaderTopNav";
 
 const MainLayout = () => {
   return (
     <>
       <SeoHelmet />
       <HeaderTopNav />
       <Outlet />
+      <EmptyFiller />
```

Comment:
mislim da nema potrebe za ovaj EmptyFiller, to vrv moze u footer da se doda ili tako nesto